### PR TITLE
Update SSM paths

### PR DESCRIPTION
**Pull request by zotoMIT**
#### Developer Checklist

- [X] The README contains the correct list of dependencies, inputs, and outputs (e.g., `terraform-docs markdown .` has been run and copied)
- [NA] Any special variables have values configured in the Terraform Cloud workspace
- [NA] Stakeholder approval has been confirmed (or is not needed)

#### What does this PR do?

These changes are required for the updated SSM parameter paths included in the template used by this repository:
https://github.com/MITLibraries/mitlib-tf-template

I update ssm_path to 3 paths, summed up in locals, which i moved to main:
* tfoutput_ssm_path = "/tfvars/${var.app_name}"
* appinput_ssm_path = "/apps/${var.app_name}"
* tfinput_ssm_path  = "/tfvars/apps-vars/${var.app_name}"

#### Helpful background context
We need to decide how to make template changes (provider tags) from the template in repositories managed by this repo.  
I think its easiest to update them by hand (there are only a few) rather than re-create.  But i'm open to suggestions.  

#### Requires Database Migrations?

NO

#### Includes new or updated dependencies?

NO

### CODE REVIEW

File: modules/tfc/variables.tf [hunk 1]
```diff
@@ -16,9 +16,6 @@ variable "environments" {
   type        = list(string)
 }
 
-locals {
-  ssm_path = "/tfvars/${var.app_name}"
-}
 
 # variable "aws_keys" {
```
Comment (cabutlermit) [inline modules/tfc/variables.tf:20]: This is probably the right time to remove these two variables (that were already commented out). I don't see any need to keep these in here any longer

**Comment by cabutlermit** [comment]
My suggestion is that the `root-budgets-prod` workspace & the `mitlib-tf-budgets` repository should be updated by hand since it's actually in use.

I also suggest that the two `apps-vars` repos/workspaces should probably be deleted and re-created since they really haven't been used other than your quick test of the stage branch.

File: modules/tfc/variables.tf (hunk 1 above)
Comment (zotoMIT) [reply modules/tfc/variables.tf]: Sure, removed.

**Comment by zotoMIT** [comment]
> My suggestion is that the `root-budgets-prod` workspace & the `mitlib-tf-budgets` repository should be updated by hand since it's actually in use.
> 
> I also suggest that the two `apps-vars` repos/workspaces should probably be deleted and re-created since they really haven't been used other than your quick test of the stage branch.

That sounds good.  I'll be interested to see what sort of job terraform does removing them.

**Comment by cabutlermit** [comment]
> > My suggestion is that the `root-budgets-prod` workspace & the `mitlib-tf-budgets` repository should be updated by hand since it's actually in use.
> > I also suggest that the two `apps-vars` repos/workspaces should probably be deleted and re-created since they really haven't been used other than your quick test of the stage branch.
> 
> That sounds good. I'll be interested to see what sort of job terraform does removing them.

I'm pretty sure that GitHub settings for the repos are "archive, don't delete" which means a `terraform destroy` followed by a `terraform apply` will fail because GitHub will still have the archived repo with the same name. If you comment out the code that creates the repos and run `terraform apply` it should also just archive the repo so an uncomment and another `terraform apply` will fail the same way. 

In my testing, I would go manually clean up the leftover archive repo in GitHub before trying to re-create.

**Review by cabutlermit — APPROVED** [review]
looks good. fingers crossed on what happens when this applies!

**Pull request merged**